sopel/modules/admin.py

```diff
@@ -247,61 +268,85 @@ def mode(bot, trigger):
     bot.write(('MODE', bot.nick + ' ' + mode))
 
 
-@sopel.module.require_privmsg("This command only works as a private message.")
-@sopel.module.require_admin("This command requires admin privileges.")
-@sopel.module.commands('set')
-@sopel.module.example('.set core.owner Me')
-def set_config(bot, trigger):
-    """See and modify values of Sopel's config object.
-
-    Trigger args:
-        arg1 - section and option, in the form "section.option"
-        arg2 - value
+def parse_section_option_value(config, trigger):
+    """Parse trigger for set/unset to get relevant config elements.
 
-    If there is no section, section will default to "core".
-    If value is None, the option will be deleted.
+    Returns:
+        - (section, section_name, static_sec, option, value) otherwise.
+            - value is `None` if omitted from command.
```

Comment:
It's me or there is an extra tabulation here?

Reply:
It is not just you.

This is removed anyway with the addition of the sphinx directives.